### PR TITLE
Use experimental paket feature to copy runtime deps

### PR DESCRIPTION
**Pull request by forki**
At the moment there is some manual work to hook the new custom task. I will make that go away when we think this is a way to go forward

### CODE REVIEW

File: src/KestrelClassic.fsproj [hunk 1]
```diff
@@ -68,6 +63,10 @@
     </Otherwise>
   </Choose>
   <Import Project="$(FSharpTargetsPath)" Condition="Exists('$(FSharpTargetsPath)')" />
+  <UsingTask TaskName="CopyRuntimeDependencies" AssemblyFile="..\.paket\paket.exe" />
+  <Target Name="BeforeBuild">
```
Comment (forki) [inline src/KestrelClassic.fsproj]: I assume we can use AfterBuild

Comment (CumpsD) [inline src/KestrelClassic.fsproj]: We have to apparently, something else cleans `build`

**Comment by forki** [comment]
PR updated

**Comment by CumpsD** [comment]
:+1:

**Comment by dsyme** [comment]
:+1:

**Pull request merged**